docs/build-instructions/windows.md

```diff
@@ -21,57 +21,47 @@ You can use either:
 Whichever version you use, ensure that:
 
  * The default installation folder is chosen so the build tools can find it
- * If using Visual Studio make sure Visual C++ support is selected/installed
- * If using Visual C++ Build Tools make sure Windows 8 SDK is selected/installed
+ * Visual C++ support is selected/installed if using Visual Studio
```

Comment:
Personally find this order less easy to skim. You have to read to the end of the line to find out if it applies to you.